### PR TITLE
[ACI] Set the ACI_EXTRA_USER_AGENT in the helm charts

### PR DESCRIPTION
**Pull request by robbiezhang**

### CODE REVIEW

File: charts/virtual-kubelet/templates/deployment.yaml [hunk 1]
```diff
@@ -60,13 +60,17 @@ spec:
           value: {{ .aciResourceGroup }}
         - name: ACI_REGION
           value: {{ .aciRegion }}
+        - name: ACI_EXTRA_USER_AGENT
+          value: {{ printf "helm-chart/aks/%s/%s" $.Chart.Name $.Chart.Version }}
```
Comment (cpuguy83) [inline charts/virtual-kubelet/templates/deployment.yaml]: This doesn't seem right?

Comment (robbiezhang) [reply charts/virtual-kubelet/templates/deployment.yaml]: anything wrong? I've verified the package locally.

Comment (cpuguy83) [reply charts/virtual-kubelet/templates/deployment.yaml]: Why are we adding aks to the generic template?

Comment (robbiezhang) [reply charts/virtual-kubelet/templates/deployment.yaml]: L45, it has the "targetAKS" flag == true

Comment (robbiezhang) [reply charts/virtual-kubelet/templates/deployment.yaml]: we will deprecate the virtual-kubelet-for-aks helm chart soon.

**Pull request merged**